### ISSUE TITLE
[Bug] IP LinkLocal v4 does not display when activated

### ISSUE THREAD

**Issue by andikay** [w2sv/WiFi-Widget#180]
### Overview

With the newest update 1.6.3 IP Link Local v4 does not show at all anymore when activated. It doesn't even show on the preview in the app.

### How to reproduce

Activate IP Link Local v4 in the Widget Configuration.

### Version

1.6.3

### Android Version

Android 14

### Device Model

Realme GT 5G

**Comment by w2sv** [comment]
I doubt this has anything to do with the new release, as I didn't touch anything having anything to do with that in the slightest. Are you sure your device actually has an ipv4 link local address, i.e., can you observe it somewhere else?

**Comment by andikay** [comment]
Thank you, you are correct, I enabled the wrong IPv4 by accident. That is embarrassing.

I have enabled the IP Site Local v4 now and it works; however, the spacing above/under it is larger than with Link Speed and RSSI, meaning the information is cut off in the 3x1 widget.

**Comment by w2sv** [comment]
The padding ain't bigger, it's just that the IP SiteLocal row is larger due to the 'SiteLocal' subscript. The vertical padding is bigger in general now compared to how it used to be, which is why the cut-off of the Link Speed row occurs.

**Comment by andikay** [comment]
Yes, could you please change it back or at least give an option so that the user can adjust it?

**Comment by w2sv** [comment]
Sure 😅

**Comment by w2sv** [comment]
You could set the font size to 'Very Small' in the meantime, that way the second row gets entirely displayed again.

**Comment by andikay** [comment]
I already have it on the smallest setting, the screenshots are with the smallest setting.

**Comment by w2sv** [comment]
Alright, on my device it works that way. Was just an idea for a temporary workaround anyways.
